colors/onedark.vim

```diff
@@ -647,6 +647,12 @@ hi link gitcommitUnmergedArrow gitcommitUnmergedFile
 
 " }}}
 
+" Vim-Specific Terminal Highlighting {{{
+"
+let g:terminal_ansi_color = [s:black.gui, s:red.gui, s:green.gui, s:yellow.gui, s:blue.gui, s:purple.gui, s:cyan.gui, s:white.gui, s:visual_grey.gui, s:dark_red.gui, s:green.gui,  s:dark_yellow.gui, s:blue.gui, s:purple.gui, s:cyan.gui, s:comment_grey.gui, s:background.gui, s:foreground.gui]
```

Comment:
```suggestion
let g:terminal_ansi_colors = [s:black.gui, s:red.gui, s:green.gui, s:yellow.gui, s:blue.gui, s:purple.gui, s:cyan.gui, s:white.gui, s:visual_grey.gui, s:dark_red.gui, s:green.gui,  s:dark_yellow.gui, s:blue.gui, s:purple.gui, s:cyan.gui, s:comment_grey.gui, s:background.gui, s:foreground.gui]
```

Reply:
Could be! I just tested it again and there is still an error. Somehow the colors only get fixed it I manually source the colors.vim file. Have to check again.

Reply:
There is no error. My setup somehow overwrites the colors...